tests/pipeline/test_alerting.py

```diff
@@ -0,0 +1,218 @@
+import random
+import pytest
+import re
+import io
```

Comment:
My linter cares about the order of these imports - it wants standard imports ("io", "re") to go before third party ones ("pytest"). The basis appears to be PEP8 Python convention. Do you care about this?

https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/wrong-import-order.html